src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -0,0 +1,140 @@
+---
+id: Code editor
+section: components
+beta: true
+cssPrefix: pf-c-code-editor
+---
+
+## Examples
+### Default
+```hbs
+{{#> code-editor}}
+  {{#> code-editor-control}}
+    {{#> code-editor-action-group}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+        <i class="fas fa-copy" aria-hidden="true"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
+        <i class="fas fa-download"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Upload code"'}}
+        <i class="fas fa-upload"></i>
+      {{/button}}
+    {{/code-editor-action-group}}
+    {{#> code-editor-tab}}
+      <i class="fas fa-code"></i>
+      {{#> code-editor-tab-text}}
+        HTML
+      {{/code-editor-tab-text}}
+    {{/code-editor-tab}}
+  {{/code-editor-control}}
+  {{#> code-editor-code}}
+    code goes here 
+  {{/code-editor-code}}
+{{/code-editor}}
+```
+
+### Overflow
+```hbs
+{{#> code-editor}}
+  {{#> code-editor-control}}
+    {{#> code-editor-action-group}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+        <i class="fas fa-copy" aria-hidden="true"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
+        <i class="fas fa-download"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Upload code"'}}
+        <i class="fas fa-upload"></i>
+      {{/button}}
+      {{#> code-editor-action-group-control}}
+        {{#> overflow-menu overflow-menu--id="overflow-menu-simple"}}
```

Comment:
Could you show an example that shows how the overflow menu is used before items have collapsed into a dropdown?

Reply:
what do you mean by this?

Reply:
I was assuming this meant the overflow menu component was being used here, but it sounds like this is just a dropdown with overflow items in it?